HelpSource/Tutorials/Getting-Started/04-Functions-and-Other-Functionality.schelp

```diff
@@ -39,49 +53,45 @@ f.value;
 
 emphasis::Different types of objects may have methods with the same name, and thus respond to the same message in different ways.::
 
 What's interesting about this is that the actual methods may differ in what they do, but as long as they implement a method with that name, they become interchangeable in your code.
 
-A good example is 'value'. All objects in SC respond to the message 'value'. When you 'call' a method, it always 'returns' something, such as a value or a result. When you call the method 'value' on a Function it will evaluate and return the result of its last line of code. The example below will return the number 5.
+A good example is "value". All objects in SC respond to the message "value". When you "call" a method, it always "returns" something. When you call the "value" method on a Function it will evaluate and return the result of its last line of code. The Function example below and will return the result on the last line (5):
 
 code::
-f = { "Evaluating...".postln; 2 + 3; };
+(
+f = {
+	"Evaluating...".postln;
+	2 + 3;
+};
 f.value;
+)
 ::
 
-Often methods simply return the object itself. This is the case with most objects and the message 'value'. The example below demonstrates this. (Everything to the right of the code:://:: is a 'comment', which means that SC just ignores it. Comments are a good idea to make your code clearer.)
-
-code::
-f = 3;			// Here I make f equal to a number
-f.value;		// Post window says: 3, i.e it returns itself
-f.value;		// Still says 3
-
-f = { 3.0.rand; };	// Here it's a Function.
-f.value;		// 3.0.rand means return a random value from 0 to 3.0 exclusive.
-f.value;		// something different
-f.value;		// something different
-::
-
-This means that by using the 'value' method Functions and other objects can be interchangeable in your code. This is an example of emphasis::polymorphism::, which is one of the powerful features of what's called Object Oriented Programming. Polymorphism just means that different objects are interchangeable (at least providing they return something sensible for what you're doing) if they respond to the same message. Object Oriented Programming (or OOP, as it's called for short) just means programming with objects. Simple, yes? Here's another short example showing this in action:
+Often methods simply return the object itself. This is the case with most objects and the message "value". The example below demonstrates this. (Everything to the right of the code:://:: is a "comment", which means that SC just ignores it. Comments are often used to leave notes on complicated or confusing pieces of code for future readers)
 
 code::
-f = { arg a; a.value + 3 };	// call 'value' on the arg; polymorphism awaits!
```

Comment:
I moved this example of polymorphism down to the bottom of this chapter b/c I felt like it made more sense to cover arguments first